app/lib/memory_repository.rb

```diff
@@ -126,6 +126,12 @@ def commit(transaction)
         rescue Repository::Conflict => e
           transaction.add_conflict(e)
         end
+      when :remove_directory
```

Comment:
will add :remove, :remove_directory in the same line both in memory and subversion. Forgot to modify

Reply:
please do the same for subversion_repository as well